.gitattributes

```diff
@@ -0,0 +1 @@
+package-lock.json   binary
```

Comment:
Never actually seen anyone do this but I think I can understand why you would want to. This rarely becomes an issue but definitely produces noise. I've seen some issues where its helpful to have line differences due to different bugs. Does it make sense to have it still be as `merge`?

Reply:
You can always override it by doing `git diff --text`.

Reply:
I'm not gonna remember to do that... Can we do something like this?
https://stackoverflow.com/questions/33085428/gitattributes-and-the-binary-option